eos-payg-csv/README.md

```diff
@@ -0,0 +1,43 @@
+eos-payg-csv
+============
+Usage:
+    eos-payg-csv ACCT_KEY_CSV_FILE
+
+Read per-device `DEVICE_ID` and Pay As You Go private keys from
+`ACCT_KEY_CSV_FILE` and generate time codes for several time periods per key.
+
+For each `DEVICE_ID`, the last corresponding key that appears in
+`ACCT_KEY_CSV_FILE` will be used to generate these unlock codes and the output
+will be `codes/DEVICE_ID.csv`. Earlier keys are ignored in case a given computer
```

Comment:
This needs to be updated now that the `codes` directory isn't created

Reply:
Fixed locally